logic.js

```diff
@@ -0,0 +1,108 @@
+
+const Tweeter = function() {
+    let _posts = [{
+        text: "First post!",
+        id: "p1",
+        comments: [
+            { id: "c1", text: "First comment on first post!" },
+            { id: "c2", text: "Second comment on first post!!" },
+            { id: "c3", text: "Third comment on first post!!!" }
+        ]
+    },
+    {
+        text: "Aw man, I wanted to be first",
+        id: "p2",
+        comments: [
+            { id: "c1", text: "Don't wory second poster, you'll be first one day." },
+            { id: "c2", text: "Yeah, believe in yourself!" },
+            { id: "c3", text: "Haha second place what a joke." }
+        ]
+    }]
+    let postIdCounter = _posts.length
+    let commentIdCounter = 1 
```

Comment:
Note that you don't use `postIdCounter` and `commentIdCounter` anywhere in your code...